dev-requirements.txt

```diff
@@ -7,42 +7,55 @@
 appdirs==1.4.4            # via virtualenv
 aspy.yaml==1.3.0          # via pre-commit
 atomicwrites==1.4.0       # via pytest
-attrs==19.3.0             # via pytest
+attrs==19.3.0             # via -c requirements.txt, jsonschema, pytest
 certifi==2020.6.20        # via -c requirements.txt, requests
 cfgv==2.0.1               # via -r dev-requirements.in, pre-commit
 chardet==3.0.4            # via -c requirements.txt, requests
 click==7.1.2              # via -c requirements.txt, flask
+configparser==4.0.2       # via -c requirements.txt, detect-secrets, importlib-metadata
+contextlib2==0.6.0.post1  # via -c requirements.txt, importlib-metadata, importlib-resources, zipp
 coverage==5.2             # via pytest-cov
 detect-secrets==0.13.0    # via -r dev-requirements.in
 distlib==0.3.1            # via virtualenv
+enum34==1.1.10            # via detect-secrets
 filelock==3.0.12          # via virtualenv
 flask==1.1.2              # via -c requirements.txt, pytest-flask
+funcsigs==1.0.2           # via pytest
+# manual edit to install only with py2
+functools32==3.2.3.post2;python_version=="2.7"  # via -c requirements.txt, detect-secrets, jsonschema
```

Comment:
There are three instances where I manually edited the generated requirements files to prevent backports from being installed when using py3.